internal/weldr/api.go

```diff
@@ -3354,7 +3408,7 @@ func (api *API) fetchPackageList(distroName string, names []string) (packages rp
 		return nil, err
 	}
 
-	solver := api.solver.NewWithConfig(d.ModulePlatformID(), d.Releasever(), api.archName, d.Name())
+	solver := api.solver.NewWithConfig(d.ModulePlatformID(), d.Releasever(), api.hostArch, d.Name())
```

Comment:
I think this should be arch now, right? Same goes for `ReposByArchName` and other `solver.NewWithConfig` uses.

Reply:
Ah indeed, I left the modules info, project depsolve and metadata preloading based on the host arch. Should I switch over one of the first two?

Reply:
I think they should all use the specific arch, it's possible there will be a different set of packages so it should be as precise as possible.